### ISSUE TITLE
Ignore CS1591 in release mode

### ISSUE THREAD

**Issue by jzabroski** [fluentmigrator/fluentmigrator#1044]
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1591

This needs to be added to each project:
```xml
<PropertyGroup Condition="'$(Configuration)'=='Release'">
    <noWarn>1591</noWarn>
</PropertyGroup>
```

**Comment by jzabroski** [comment]
The purpose of this is to eliminate >8000 lines of warnings in the Travis CI output, which causes us to have to download the raw log to view it.

Long term we will add it back, but there are so many warnings right now, nobody will be motivated to fix it.

**Comment by eloekset** [comment]
@jzabroski, should this issue be reopened? 

I was about to create a new one about the endless build warnings for CS1591, but then I found this instead. The first PR referenced here probably fixes the issue for one of the projects, but still all other projects must get the same noWarn propert set?

This is an example from a recent build log:
![image](https://user-images.githubusercontent.com/791721/67308038-c3c4cf00-f4f9-11e9-8f4c-09bdab381ae5.png)

**Comment by jzabroski** [comment]
We should just GhostDoc the errors.

**Comment by eloekset** [comment]
GhostDoc looks like a great tool for this job. Do you have a license for Pro already? From this [features comparison](https://submain.com/products/ghostdoc.aspx?utm_campaign=ghostdoc&utm_medium=listing&utm_source=visualstudiogallery), it seems like we should have access to a Pro license.

**Comment by eloekset** [comment]
I tested a trial of GostDoc Pro on all projects using default settings. It does way too much - overwrites existing good comments and adds an additional license header at the top of each file.

The build log gets rid of all that warning noise though, so that's good to have confirmed. Only 5 warnings left.

I won't send a PR of it. We'll have to find out how to configure it so that existing comments are not overwritten, and make it skip the license header.

**Comment by jzabroski** [comment]
I believe GhostDoc Pro has a Control+Shift+. keyboard shortcut or similar that only fixes the guilty method you are focused on.

I last used GhostDoc Pro about 5 years ago while I worked as a contractor at a major place that mandated everything was commented.

**Comment by eloekset** [comment]
Yes, we can definitely use it manually on just the methods we want. But even that takes a lot of effort, since we have tons of code without XML comments.

The Pro version supports automatically commenting all code in a project. I'll take a look at the docs if it's possible to configure to just add comments where it's missing.
